warehouse/query-core/src/main/java/datawave/query/attributes/ExcerptFields.java

```diff
@@ -59,7 +59,8 @@ public static ExcerptFields from(String string) {
         String[] fieldParts = string.split(Constants.COMMA);
         for (String fieldPart : fieldParts) {
             String[] parts = StringUtils.split(fieldPart, Constants.FORWARD_SLASH);
-            excerptFields.put(parts[0], Integer.valueOf(parts[1]));
+            String direction = parts.length == 3 ? parts[2] : "both";
```

Comment:
Minor quip, but it might be worth considering putting the "both" string as a constant for the two spots in this class where its used. Here and on line 140

Reply:
Good catch, will update